examples/print_vulnerability_affected_projects.py

```diff
@@ -19,23 +19,27 @@
 parser.add_argument("vulnerability", help="A CVE or BDSA number, e.g. CVE-2016-4009")
 args = parser.parse_args()
 
-affected_projects = hub.get_vulnerability_affected_projects(args.vulnerability.upper())
+affected_projects = hub.get_vulnerability_affected_projects(args.vulnerability.upper()).get('items', [])
 
-if 'totalCount' in affected_projects and affected_projects['totalCount'] > 0:
+if affected_projects:
 	ttable = [[
 		"project-name", 
 		"version", 
 		"phase", 
 		"distribution", 
-		"last-bom-update", 
+		"scan-info", 
 		"Owner Name", 
 		"Owner email"]]
 
-	for affected_project in affected_projects['items']:
+	for affected_project in affected_projects:
 		# Get the Owner info for the project
-		project_json = hub.get_project_by_id(affected_project['project']['id'])
-		if 'projectOwner' in project_json:
-			owner_response = hub.execute_get(project_json['projectOwner'])
+		# project_json = hub.get_project_by_id(affected_project['project']['id'])
+		project_url = affected_project['project']
+		custom_headers = {'Accept': 'application/vnd.blackducksoftware.project-detail-4+json'}
+		project_details = hub.execute_get(project_url, custom_headers=custom_headers).json()
+		if 'projectOwner' in project_details:
+			custom_headers = {'Accept': 'application/vnd.blackducksoftware.user-1+json'}
```

Comment:
I think this should probably use the public V4 media type instead: application/vnd.blackducksoftware.user-4+json

Reply:
Thanks, I updated the media type.